tests/std/tests/P0896R4_ranges_alg_generate/test.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <concepts>
+#include <ranges>
+
+#include <range_algorithm_support.hpp>
+
+using namespace std;
+

```

Comment:
Last minute note: Should these tests also "Validate dangling story" as many [others ](https://github.com/microsoft/STL/blob/master/tests/std/tests/P0896R4_ranges_alg_is_sorted/test.cpp#L17-L19) do?  Or is there reason these algorithms should be exempt?

Reply:
Yes! `generate_n` doesn't accept `range` arguments, but `generate` does so it needs some `dangling` coverage.